client/src/modules/stock/inventory-adjustment/inventory-adjustment.html

```diff
@@ -0,0 +1,102 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.STOCK</li>
+      <li ng-class="{ 'title' : !StockCtrl.depot.uuid, 'static' : StockCtrl.depot.uuid }">
+        <span translate>INVENTORY_ADJUSTMENT.TITLE</span>
+      </li>
+      <li class="title" ng-if="StockCtrl.depot.uuid">
+        <span>{{ StockCtrl.depot.text }}</span>
+      </li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <bh-dropdown-menu>
+          <bh-dropdown-menu-item>
+            <bh-change-depot on-select="StockCtrl.onChangeDepot(depot)"></bh-change-depot>
+          </bh-dropdown-menu-item>
+        </bh-dropdown-menu>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- content -->
+<div class="flex-content">
+  <div class="container-fluid">
+    <form name="StockForm" bh-submit="StockCtrl.submit(StockForm)" novalidate>
+      <div class="row">
+        <div class="col-xs-12 col-sm-6 col-md-4">
+
+          <!-- date -->
+          <bh-date-editor
+            date-value="StockCtrl.movement.date"
+            on-change="StockCtrl.onDateChange(date)">
+          </bh-date-editor>
+
+          <!-- note -->
+          <div class="form-group"
+            ng-class="{ 'has-error' : StockForm.$submitted && StockForm.description.$invalid }">
+            <label class="control-label">
+              <span translate>FORM.LABELS.DESCRIPTION</span>
+            </label>
+            <textarea
+              class="form-control"
+              name="description"
+              ng-model="StockCtrl.movement.description"
+              translate-attr="{ placeholder : 'FORM.PLACEHOLDERS.DESCRIPTION' }"
+              ng-maxlength="StockCtrl.maxLength"
+              required>
+            </textarea>
+            <div class="help-block" ng-messages="StockForm.description.$error" ng-show="StockForm.$submitted">
+              <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+            </div>
+          </div>
+        </div>
+        <div class="col-xs-12 col-sm-6 col-md-6">
+          <h3 translate>INVENTORY_ADJUSTMENT.TITLE</h3>
+          <p translate>INVENTORY_ADJUSTMENT.DESC</p>
+        </div>
+      </div>
+
+      <!-- "Add number of grid rows" input-group -->
+      <div class="row" style="padding-bottom : 5px;">
```

Comment:
If we load all stock at the beginning of the operation, do we still need this input group?